### PR TITLE
Enabled publishing to NPM registry

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "unobtrusive",
+    "ajax",
+    "mvc",
+    "asp.net",
+    "jquery"
+  ],
+  "author": "Microsoft",
+  "license": "https://www.microsoft.com/web/webpi/eula/aspnetcomponent_rtw_enu.htm",
```
Comment (Eilon) [inline package.json]: Hmm, is this the EULA we have on other JS files or something? I think we want to usehttps://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm

E.g. see what we do in Coherence-Signed for all the NUPKGs: https://github.com/aspnet/Coherence-Signed/blob/dev/dnx.msbuild#L682

Do we need to go update other JS files?

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "jquery-ajax-unobtrusive",
-    "version": "3.2.4",
+    "version": "3.2.5",
```
Comment (Eilon) [inline bower.json]: Isn't this still 3.2.4? We haven't "released" 3.2.4 yet, no?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "version": "3.2.5",
```
Comment (Eilon) [inline package.json]: See above.

File: bower.json (hunk 2 above)
Comment (kichalla) [inline bower.json]: Yeah, I agree. We should only update versions when there is actual change to the .js file logic.

File: package.json (hunk 1 above)
Comment (kichalla) [inline package.json]: I picked up the license details link from the existing nuget package available on nuget.org. Sure will change it.

You mean updating the JS files' headers. How should the content look like? should it be the standard .NET foundation header?

Comment (Eilon) [inline package.json]: The JS files are fine, it's just the wrong license that we link to.

File: bower.json (hunk 2 above)
Comment (Eilon) [inline bower.json]: Not just that - we actually haven't shipped 3.2.4 yet - it's not yet tagged.

**Comment by Eilon** [comment]
:watch:

File: bower.json (hunk 2 above)
Comment (kichalla) [inline bower.json]: Ah, good point.

**Comment by kichalla** [comment]
:up: :date: I am planning to push the tag against this merged PR's commit.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "unobtrusive",
+    "ajax",
+    "mvc",
+    "asp.net",
+    "jquery"
+  ],
+  "author": "Microsoft",
+  "license": "http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm",
```
Comment (Eilon) [inline package.json]: Looks like this URL auto-redirects to HTTPS anyway, so we should absolutely just use that.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "version": "3.2.4",
+  "description": "Add-on to jQuery Ajax to enable unobtrusive options in data-* attributes",
+  "main": "jquery.unobtrusive-ajax.js",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/aspnet/jquery-ajax-unobtrusive"
```
Comment (Eilon) [inline package.json]: HTTPS

**Comment by Eilon** [comment]
:+1: to pushing the tag as well.

**Comment by kichalla** [comment]
:up: :date: Fixed the urls to use https

**Comment by Eilon** [comment]
Still one HTTPS missing.

**Comment by kichalla** [comment]
That url was generated by `npm init` and probably it doesn't hurt to change it to `https` but I noticed this is different than what we have for `jquery-validation-unobtrusive` where it uses `git://` instead. https://github.com/aspnet/jquery-validation-unobtrusive/blob/master/package.json#L9

File: package.json (hunk 5 above)
Comment (Eilon) [inline package.json]: ?

**Comment by kichalla** [comment]
:up: :date:

**Comment by Eilon** [comment]
https://:shipit::/

**Pull request merged**